src/main/resources/script_templates/ImageJ2-Scripts/Apply_Threshold.py

```diff
@@ -0,0 +1,18 @@
+# @ImageJ ij
+# @Dataset data
+ 
+method_threshold = "otsu"
+ 
+# Get the histogram
+histo = ij.op().run("image.histogram", data.getImgPlus())
+ 
+# Get the threshold
+threshold = ij.op().run("threshold.%s" % method_threshold, histo)
+ 
+# Apply the threshold
+thresholded = ij.op().run("threshold.apply", data.getImgPlus(), threshold)
+ 
+# Create output
+thresholded = ij.dataset().create(thresholded)
+ 
+ij.ui().show("thresholded", thresholded))
```

Comment:
I just tried to run this script.  It failed, and I noticed there is an extra ```)``` at the end of the last line.